dist/object.js

```diff
@@ -192,6 +202,8 @@ function getCached(name) {
 function defineProp(object, name) {
   var opts = arguments.length > 2 && arguments[2] !== undefined ? arguments[2] : {};
 
+  validateOptions(opts);
```

Comment:
This might be a good thing to only check in non-production, non-minified builds.  Would have to check the webpack setup, but typically you would use:
```if (process.env.NODE_ENV !== "production")``` and provide that value via webpack, and minification JS would strip that code post build if it results to a definitely false predicate: `"production" !== "production"`

Reply:
Thanks, I think I will add this.

We are also thinking of making it a warning for now. With a note to make it an error next timet the version is bumped.

Reply:
@alexraginskiy Good idea. There are likely several more error case checks that could be wrapped in an env check.

@BrianMehrman If you can get the env check working, then I'd prefer to have it throw an exception when an invalid option is detected.

Reply:
I can add the environment check and throw an exception if we are fine with this change breaking things for non production environments.

Reply:
Added the check for the production environment here. [6d9339afab6](https://github.com/centro/transis/pull/52/commits/6d9339afab67cc39e7090ba25ea192d997f97e19)